atuin/src/command/client/search/interactive.rs

```diff
@@ -536,7 +537,15 @@ pub async fn history(
             atuin_client::settings::Style::Compact => true,
             atuin_client::settings::Style::Full => false,
         };
-        terminal.draw(|f| app.draw(f, &results, compact, settings.show_preview))?;
+        terminal.draw(|f| {
+            app.draw(
+                f,
+                &results,
+                compact,
+                settings.show_preview,
```

Comment:
I think at this point it is probably sensible to pass a reference to settings, rather than pass in some parts of it